.github/workflows/frontend-verify.yml

```
@@ -33,3 +33,23 @@ jobs:
           REACT_APP_BUILD_TIMESTAMP=$(date +%s) 
           REACT_APP_DEV_HINT=true
           npm run build 
+  
+  e2e-test:
+    needs: "lint-and-compile"
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18.10.0'
```

```suggestion
      - uses: actions/setup-node@v4
        with:
          node-version: '18'
```

Node v18.x for less maintenance effort at the cost of potential differences. In theory our frontend should be using the latest v18.x as well for the build process